thirdparty/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 include (Compiler)
 
-if (BOND_ENABLE_GRPC)
+if (BOND_ENABLE_GRPC AND NOT BOND_SKIP_GRPC_LIB_COMPILE)
```

Comment:
Doesn't this mean both `BOND_ENABLE_GRPC` should be enabled and `BOND_SKIP_GRPC_LIB_COMPILE` be disabled?

Seems like the best thing to do would be to set `BOND_ENABLE_GRPC` to `FALSE` and let the user opt-in?